docs/api/migration-guides/local-simulators.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Migrate to local simulators
+description: Migrate from using the IBM Quantum cloud simulators (such as ibmq_qasm_simulator) to using local simulators.
+---
+
+# Migrate to local simulators
+
+In quantum computing, the choice between using simulators and quantum hardware is crucial for making progress in the field. While simulators are useful for testing and debugging, in this era of quantum utility, quantum development and industry advancement requires actual hardware. As part of the move to quantum utility, IBM Quantum&trade; will retire its cloud simulators on 15 May 2024. This guide explains in more detail why this is happening, and how to change from cloud to local simulators. 
+
+## Why are the cloud simulators being retired?
+
+The cloud simulators are being retired for several reasons:
+
+### Simulators have limitations
```

Comment:
This section about why the cloud simulators are being retired feels very long, should we turn each of the "reasons" a twisty instead? so people can more quickly get to the code changes if they don't care about the whys